### DISCUSSION TITLE
Update users password

### DISCUSSION THREAD

**Discussion by Sivolen** [nilsteampassnet/TeamPass#4018] [General]
Hello!
When updating I get this message "Users password not encrypted with new library. Is a blocker to upgrade to 3.1.3"
Can you tell me how to generate passwords for all users using the new library?

**Comment by nilsteampassnet** [comment]
@Sivolen 
The message indicates a blocker for release 3.1.3.
We are currently in 3.1.1, so not an issue, just a warning.
I will change the symbol to make it less "agressive"

**Reply by nilsteampassnet** [reply]
Please see #4020
